frontend/src/static/js/v_summary.js

```diff
@@ -389,9 +391,37 @@ window.vSummary = {
 
       this.filtered = full;
     },
+    renderAuthorShipTabHash() {
```

Comment:
this could be place in main.js, and it makes more sense since the main app should be the one doing all the view handling.

We make use of vue event emitting if have trouble tracing, it should be what you are googling. Hahhaha

Reply:
I'm having difficulties passing `minDate` and `maxDate` (when it becomes available) from `vSummary` back to `main.js`, to generate `infoHash` for the authorship tab. 

Is using `EventBus` a way to get around this?

Reply:
@ongspxm any updates on this?

Reply:
wait, authorship tab no need `minDate`, `maxDate` right? our data also cannot sort by date

Reply:
in my `zoomin` pr, im gonna create some sort of method to pass the date out to the tab view (not authorship), if im right, authorship dun really need

Reply:
![image](https://user-images.githubusercontent.com/30564248/53298420-62603980-3860-11e9-800d-a932cecfbd11.png)
`minDate` and `maxDate` are needed here